airbyte-lib/airbyte_lib/source.py

```diff
@@ -168,7 +170,7 @@ def check(self):
     def install(self):
         self.executor.install()
 
-    def _read(self) -> Iterable[AirbyteRecordMessage]:
+    def _read(self, target: str) -> Iterable[AirbyteRecordMessage]:
```

Comment:
It's not clear to me what 'target' means in this context. Can we add a docstring to explain it?

Reply:
It's about where the data is loaded to - a cache or streamed directly via iterable.